terraform/modules/gro_ingestion_service/lambdas/src/models/GroXmlFile.ts

```diff
@@ -0,0 +1,6 @@
+import { GroDeathRegistration } from "./GroDeathRegistration"
+
+export interface GroXmlFile {
```

Comment:
This isn't the XML file, this is the JSON object of it, could you pls try a rename (naming is hard :( )

Reply:
It's not needed anymore whoop